pkgs/applications/networking/browsers/chromium/update.nix

```diff
@@ -51,6 +51,11 @@ in rec {
       urls = mkUrls "i386";
       sha256 = chanAttrs.sha256bin32;
     } else throw "No Chrome plugins are available for your architecture.");
+
+    toolchain = import ./toolchain.nix {
```

Comment:
So you're fetching the source to `$out` of the `toolchain.nix`, which basically implies that you need to "turn off sandboxing temporarily to fetch the toolchain" and afterwards turn on to build it along with the browser itself, do I understand this correctly?

If yes, this shouldn't be the way to go but we should rather look into what the toolchain fetcher is doing and do the packaging in a way it doesn't require sandboxing to be disabled.